tutorials/fiori-tools-mockserver-opa-testing/fiori-tools-mockserver-opa-testing.md

```diff
@@ -0,0 +1,606 @@
+                    });
+                }
+            };
+            return Journey;
+        });
+        ```
+
+        >In line 9 of the coding fragment above, the application is started using the function `iStartMyApp()`. Please add the name (semantic object and action) as a parameter to the function `iStartMyApp("travellist-tile")`. You can find the name of your application in the file **/webapp/test/flpSandbox.html** in the `applications:` parameter, which should look similar to the one listed below. Line 3 contains the name of the application.
+
+
+        ```flpSandbox.html
+        ...
+          applications: {
+    				"travellist-tile": {
+    					title: "Travel Management",
+    					description: "Fiori elements",
+    					additionalInformation: "SAPUI5.Component=sapuidemo.travellist",
+    					applicationType: "URL",
+    					url: "../"
+    				}
+    			}
+        ...
+        ```
+
+
+      - `pages/MainListReport.js`, `pages/MainObjectPage.js`
+
+        These are the files where the test library is included (see sap.ui.define statement) and where you can implement application-specific actions and assertions.
+
+        ```javascript
+        sap.ui.define(['sap/fe/test/ListReport'], function(ListReport) {
+            'use strict';
+
+            var AdditionalCustomListReportDefinition = {
+                actions: {},
+                assertions: {}
+            };
+
+            return new ListReport(
+                {
+                    appId: 'sap.fe.demo.travellist',
+                    componentId: 'TravelList',
+                    entitySet: 'Travel'
+                },
+                AdditionalCustomListReportDefinition
+            );
+        });
+        ```
+
+3. The `JourneyRunner` class
+
+    This class is used to simplify the start of OPA journeys and to control central OPA5 configuration parameters. An instance of the class is created in the file `Opa.qunit.js` (see chapter 2 in this step). The instance defines the application and the page objects which are used for the OPA test, and the `run` method of the class executes the journey.
+
+    Please see further options of this class in the SAPUI5 documentation under
+
+    ```URL
+    https://sapui5.hana.ondemand.com/#/api/sap.fe.test.JourneyRunner
+    ```
+
+
+[VALIDATE_2]
+[ACCORDION-END]
+
+
+[ACCORDION-BEGIN [Step 3: ](Start the tests)]
+
+To start the OPA journey `OpaJourney.js`, you can choose from one of the following options:
+
+- Start via Script from the file `package.json`
+
+    RAP-based applications like the one used for this tutorial, typically provide a start-script command, which is available in the `package.json` file.
+    Check the file for an entry like
+    ```package.json
+    "scripts": {
+      ...
+      "int-tests": "fiori run --config ./ui5-mock.yaml --open 'test/integration/opaTests.qunit.html'"
+      ...
+    }
+    ```
+    If this entry is available, you can start the tests using
```

Comment:
Please do not use notes within a sentence. Please rephrase.

Reply:
done